### DISCUSSION TITLE
[2024-10-25] Meeting minutes

### DISCUSSION THREAD

**Discussion by lucamolteni** [apache/incubator-kie-issues#1577] [Weekly Meetings]
- Current state of CI
  * DDoS due to the moving of Kogito Images
  * Nobody knows anything about the current status DDoS 
* Python Failures
  *  It should be fixed
- Apache KIE 10 release
  * https://github.com/apache/incubator-kie-kogito-apps/pull/2124 (merged)
  * Tiago on Zulip: "I won't trigger an ad-hoc build, and will wait for a new timestamped SNAPSHOT to be triggered this Sunday. Will monitor to see how it goes."
  * This is blocking the release and needs reviewing https://github.com/apache/incubator-kie-kogito-runtimes/pull/3724
    * @mcruzdev will write to the ML to speak about the problem
    * In the meanwhile a PR removing the test will be created
  * Removed the productised profile from OP as well https://github.com/apache/incubator-kie-optaplanner/pull/3136

**Comment by tiagobento** [comment]
@lucamolteni Sorry I couldn't attend the meeting today and didn't find a link to the recording. Can you please share more details about this part?
> - Current state of CI
>   - DDoS due to the moving of Kogito Images
>   - Nobody knows anything about the current status DDoS

Also, regarding my message about timestamped SNAPSHOTs, it is related to the `main` stream, so no intersection with the 10.0.x stream, nor the Apache KIE 10 release.

**Reply by lucamolteni** [reply]
Thanks for the explanation @tiagobento - I don't have the link for the recording as I've never received any email, perhaps it's the organiser receiving the video? @porcelli ^
